dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/util/UNFUtilTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2020. BEST S.A. and/or its affiliates. All rights reserved.
+ */
+package edu.harvard.iq.dataverse.util;
+
+import edu.harvard.iq.dataverse.ingest.IngestUtilTest;
+import org.dataverse.unf.UNFUtil;
+import org.dataverse.unf.UnfException;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class UNFUtilTest {
+
+	@Test
+	public void calculateUNF() {
+		String[] unfValues = {"a", "b", "c"};
+		String datasetUnfValue = null;
+		try {
+			datasetUnfValue = UNFUtil.calculateUNF(unfValues);
```

Comment:
This is not our class :P

Reply:
Oh jesus I just noticed they are testing it. I think this whole test should be removed, it's not like we can fix the code if Harvard changes something.

Reply:
Removed